### DISCUSSION TITLE
close old account that i lost acces to

### DISCUSSION THREAD

**Discussion by nakibrayane** [community/community#54615] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i once owned the account https://github.com/nakibrayan but i lost my password and email because i didn't backup my passwords database and i had a hard drive failure in my computer.

so there is no way for me to login to that account.

i don't care about it, i just want to use the same username i had once.

is there a way for me to close that old account and make it's username free to use knowing that the it is inactive.

sorry for the bad English :)

**Comment by hrishikesh-kocharekar-dev** [comment, marked as answer]
Hi Rayan, If you no longer have access to your old GitHub account, and you want to use the same username again, you can try contacting GitHub Support to request that they free up the username for you. Here's how you can contact them:

Go to https://support.github.com/contact.
Select "Your account" and then "I need to change my username".
Choose "I can't access my account" and then "No" when asked if your issue was resolved by any of the suggested articles.
Fill out the form with your name, email, and a detailed explanation of your situation, including your old username and the reason why you lost access to the account.
Submit the form and wait for a response from GitHub Support.
Keep in mind that GitHub's username policy requires that usernames cannot be used for impersonation or trademark infringement, and there is no guarantee that they will free up the username for you.
